Sources/SwiftGenKit/Parsers/CoreData/CoreDataParser.swift

```diff
@@ -41,7 +41,7 @@ public enum CoreData {
       self.warningHandler = warningHandler
     }
 
-    public static let defaultFilter = "[^/]\\.xcdatamodeld?$"
+    public static let defaultFilter = filterRegex(for: ["xcdatamodeld?"])
```

Comment:
if we apply `NSRE.escape` don't forget to also change this to `"xcdatamodel", "xcdatamodeld"`